client/src/pages/QuizPage/index.js

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from 'react'
+import Swal from 'sweetalert2'
+import RadioGroup from '@material-ui/core/RadioGroup'
+import FormControlLabel from '@material-ui/core/FormControlLabel'
+import Radio from '@material-ui/core/Radio'
+import * as yup from 'yup'
+import Button from '../../components/Button'
+
+import './style.css'
+
+ function QuizPage (props) {
+
+    const [trusteesValue, setTrusteesValue] = useState()
+    const [charityValue, setCharityValue] = useState()
+    const [annualTurnover, setAnnualTurnover] = useState()
+    const schema = yup.object().shape({
+      trusteesValue: yup.string().required(),
+      charityValue: yup.string().required(),
+      annualTurnover: yup.number().required(),
+    })
+    const requiredSchema = yup.object().shape({
+      trusteesValue: yup.string().matches(/Yes/),
+      charityValue: yup.string().matches(/Yes/),
+      annualTurnover: yup.number().min(50000),
+    })
+
+    const  handleTrusteesChange = (event) => {
+      setTrusteesValue(event.target.value);
+    }
+
+    const  handleCharityChange = (event) => {
+      setCharityValue(event.target.value);
+    }
+
+    const handleAnnualTurnover = (event) => {
+      setAnnualTurnover(event.target.value)
+    }
+
+    const handleSubmit = () => {
+      schema.validate({ trusteesValue, charityValue, annualTurnover}).catch(function(err) {
+        if(err) {
+          Swal.fire({
+            type: 'error',
+            title: 'Oops...',
+            text: err.errors
+          })
+        }
+      })
+
+      requiredSchema
+      .isValid({
+        trusteesValue, 
+        charityValue,
+        annualTurnover
+      })
+      .then(function(valid) {
+        //valid is boolean, when valid === true, then go to the QuestionsPage
```

Comment:
we can use the handleStaticButton as long as long as we pass it as a prop. Good job.